### PR TITLE
test: Update pg log test for new trimming behavior

### PR DESCRIPTION
**Pull request by dzafman**
Fixes: https://tracker.ceph.com/issues/43864

Signed-off-by: David Zafman <dzafman@redhat.com>

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by dzafman** [comment]
@jdurgin @neha-ojha The expected log sizes are just what resulted from the current code.  I'm just. assuming they are correct.  I don't know why it alternates between 3 and 4 at the end of the last test.

File: qa/standalone/osd/repro_long_log.sh [hunk 1]
```diff
@@ -116,33 +118,34 @@ function TEST_trim_max_entries()
 
     setup_log_test $dir || return 1
 
-    ceph tell osd.\* injectargs -- --osd-min-pg-log-entries 1
+    ceph tell osd.\* injectargs -- --osd_target_pg_log_entries_per_osd 2 || return 1
+    ceph tell osd.\* injectargs -- --osd-min-pg-log-entries 2
     ceph tell osd.\* injectargs -- --osd-pg-log-trim-min 2
     ceph tell osd.\* injectargs -- --osd-pg-log-trim-max 4
 
     # adding log entries, should only trim 4 and add one each time
```
Comment (dzafman) [inline qa/standalone/osd/repro_long_log.sh]: Well this is an incorrect comment

**Comment by neha-ojha** [comment]
retest this please

**Comment by dzafman** [comment]
jenkins test make check

**Pull request merged**